### DISCUSSION TITLE
Offline behavior

### DISCUSSION THREAD

**Discussion by daudrain** [matomo-org/matomo-sdk-ios#457] [Q&A]
Hi,

Does this SDK support offline mode? Meaning the tracked data are stored on disk before being sent to the Matomo server.

I am seeing [PR 377](https://github.com/matomo-org/matomo-sdk-ios/pull/377) and [PR 408](https://github.com/matomo-org/matomo-sdk-ios/pull/408) that look to provide such support. Is there any plan to merge one of them?

Thank you,
David

**Comment by brototyp** [comment]
Hi David,

Thanks for getting in touch. #408 was merged in the meantime. Feel free to try it out. And please get back with any feedback on it. 

Best, Cornelius
